Makefile

```diff
@@ -27,21 +27,24 @@ endif
 .PHONY: test-unit
 
 images:
-	imagebuilder -f Dockerfile -t openshift/origin-cluster-kube-apiserver-operator .
+	imagebuilder -f Dockerfile -t $(DOCKER_ORG)/origin-cluster-kube-apiserver-operator .
 .PHONY: images
 
 clean:
 	$(RM) ./cluster-kube-apiserver-operator
 .PHONY: clean
 
+, := ,
+IMAGES ?= cluster-kube-apiserver-operator
+QUOTED_IMAGES=\"$(subst $(,),\"$(,)\",$(IMAGES))\"
+
 origin-release:
```

Comment:
not excited about this chain, but I get why - let me see if I can come up with something cleaner

Reply:
also given part of it is preexisting you don't actually need to wait